Sources/Client.swift

```diff
@@ -234,25 +234,25 @@ extension Client {
     /**
      Fetch a single Asset from Contentful
 
-     - parameter identifier: The identifier of the Asset to be fetched
+     - parameter id: The id of the Asset to be fetched
```

Comment:
Hmm, can we explain what `id` means, aka replace the 2nd `id` with `identifier` again? (Like you did with all the other refs?)